### PR TITLE
feat(a11y): Accessibility improvements in several components

### PR DESCRIPTION
**Pull request by pladaria**

### CODE REVIEW

File: src/highlighted-card.tsx [hunk 1]
```diff
@@ -70,7 +70,7 @@ const Dismissable: React.FC<DismissableProps> = ({children, onClose = () => {}})
     const {colors, texts} = useTheme();
 
     return (
-        <div className={classes.dismissableContainer}>
+        <aside className={classes.dismissableContainer}>
```
[the remark this answers is not in the document]
Comment (pladaria) [reply src/highlighted-card.tsx:73]: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/aside

> The <aside> HTML element represents a portion of a document whose content is only indirectly related to the document's main content. Asides are frequently presented as sidebars or call-out boxes.

The dismissable card is like a callout

File: src/callout.tsx [hunk 2]
```diff
@@ -46,15 +46,27 @@ type Props = {
...
+    buttonLink,
+    'aria-label': ariaLabel,
+}) => {
     const needsButtonLinkAlignment = buttonLink && !button;
     const isInverse = useIsInverseVariant();
     const classes = useStyles({needsButtonLinkAlignment});
     const {colors, texts} = useTheme();
     return (
-        <section role="alert" className={classNames(classes.container, {[classes.overInverse]: isInverse})}>
+        <aside
```
[the remark this answers is not in the document]
Comment (pladaria) [reply src/callout.tsx:66]: see other comment

File: src/card.tsx [hunk 3]
```diff
@@ -167,11 +170,12 @@ export const MediaCard: React.FC<MediaCardProps> = ({
     body,
     button,
     buttonLink,
+    'aria-label': ariaLabel,
 }) => {
     const classes = useMediaCardStyles({media});
     return (
         <Boxed className={classes.boxed}>
-            <article className={classes.mediaCard}>
+            <article className={classes.mediaCard} aria-label={ariaLabel ?? title}>
```
[the remark this answers is not in the document]
Comment (pladaria) [reply src/card.tsx:178]: checked with chrome's screen reader extension but I get unwanted results in android, will investigate that

File: src/badge.tsx [hunk 4]
```diff
@@ -55,9 +64,11 @@ const Badge: React.FC<Props> = ({children, value, 'aria-label': ariaLabel}) => {
     }
 
     return (
-        <div className={classes.container} aria-label={ariaLabel}>
+        <div className={classes.container}>
             {value ? (
                 <div
+                    role="presentation"
+                    aria-hidden="true"
```
Comment (atabel) [inline src/badge.tsx:71]: isn't the aria-hidden redundant with role presentation? Why do we need both?

**Review by atabel — APPROVED** [review]

File: src/card.tsx (hunk 3 above)
[the remark this answers is not in the document]
Comment (pladaria) [reply src/card.tsx]: for the record: android screen reader is weird, on OSX's voiceover works as expected

File: src/badge.tsx (hunk 4 above)
Comment (pladaria) [reply src/badge.tsx:71]: voiceover was reading it even with role="presentation" :/

**Pull request merged**